examples/server_updating.py

```diff
@@ -42,20 +43,38 @@
 
 
 async def updating_task(context):
-    """Run every so often and update live values of the context.
+    """
+    This subroutine will run continously beside the server and
+    increment each two seconds some values (via asyncio.create_task
+    in run_updating_server).
 
-    It should be noted that there is a race condition for the update.
+    It should be noted that getValues and setValues are not thread safe.
     """
-    _logger.debug("updating the context")
+
     fc_as_hex = 3
     slave_id = 0x00
-    address = 0x10
-    values = context[slave_id].getValues(fc_as_hex, address, count=5)
-    values = [v + 1 for v in values]  # increment by 1.
-    txt = f"new values: {values!s}"
-    _logger.debug(txt)
+    address = 7
```

Comment:
You are changing the addresses I hope you tested with our standard server. We use the same addresses for all examples.